.env

```
@@ -6,6 +6,8 @@ LOG_LEVEL=info
 ZIPKIN_URL=http://localhost:9411
 MONGODB_URI=mongodb://admin:admin@localhost:27017/monitoring?authSource=monitoring
 
+EXTERNAL_DATA_PROCESSOR=http://localhost:8080
```

```suggestion
EXTERNAL_DATA_PROCESSOR=http://localhost:8000
```
Change the default port of the data processor since it collides with the local Kubernetes proxy port.
Let's use `8000` for now.